src/lib/parse-cdn.ts

```diff
@@ -45,6 +48,12 @@ export const getCDN = (): string => {
   }
 }
 
+export const getNextIntegrationsURL = () => {
+  const cdn = getCDN()
+  const path = `${cdn}/next-integrations`
```

Comment:
```suggestion
  return `${cdn}/next-integrations`
```

Interesting, I am curious to learn what made you decide to use anchor variables style rather than inlining the return 🤔 Nothing here btw, I am just curious of the habit.

Reply:
That’s a good question!

In part, it was copypasta from the refactor. 

On the broad topic, I’m pretty inconsistent, but I do feel that raw return make debugging harder for “complex” expressions like “return pipe(doFoo, doBar)”, since setting a breakpoint to capture a function’s return value can be tricky if not impossible for many debuggers, so you often have to go to the caller to set the breakpoint. I have no hard and fast rules — I think intermediate variables are one of those things that I don’t attach hard and fast rules to, it’s a tool to communicate developer intent.

Here, I think we could definitely do without the return, as it’s quite simple.

Reply:
> On the broad topic, I’m pretty inconsistent, but I do feel that raw return make debugging harder for “complex” expressions like “return pipe(doFoo, doBar)”,

Right right, I can see that some debuggers allow you to step into the return statement itself, while the part is to step out and see the value there rather than caring at the function call level ... 

I see I see ... just trying to learn about you in general.

Thank you for responding thou 💜